### ISSUE TITLE
Creating the GCP project in "Location: No organization" is somewhat insecure

### ISSUE THREAD

**Issue by wildbrain-tech** [MicrosoftDocs/CloudAppSecurityDocs#182]
Creating the GCP project in "Location: No organization" is somewhat insecure, as it means an independent Google user account created outside and NOT administered as part of the the G Suite organization essentially can take full control over your G Suite account via the API. That account is not subject to any of the organization's security policies, and could just be any old Gmail account, such as personal account of the dev who set up the integration in the first place. 
Since you're linking the project to a G Suite org anyways, its FAR better to create the project from a Google account created and administered from within the org, then set the Location to Your Organization Name. From there, you can also set your Oauth consent authentication from "Public" to "Internal", since the service account you're using is now in your domain. This is an additional security benefit. 
Also this guide recommends adding the G Suite API scope of  "View and Manage the provisioning of users on your domain". Why does it need such a high level of access if this is for primarily a monitoring tool? View, sure, but provision? 
For users who have followed this guide to a T, I recommend including adding an Organizational G Suite account as an owner to the project, migrating it into your org, then removing the 'personal' Google account that created the project in the first place.
Slightly ironic that a guide for a security monitoring tool is potentially opening a HUGE backdoor into the org you're trying to secure in the first place!
TLDR: MS, update your guide to creating the GCP project WITHIN the org the admin is trying to secure. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a1f41e1e-2541-8773-efc2-eeb83a403a15
* Version Independent ID: 2f2d0316-9e75-44d6-0ab6-a56b1352a1e2
* Content: [Connect G Suite to Cloud App Security](https://docs.microsoft.com/en-us/cloud-app-security/connect-google-apps-to-microsoft-cloud-app-security#feedback)
* Content Source: [CloudAppSecurityDocs/connect-google-apps-to-microsoft-cloud-app-security.md](https://github.com/Microsoft/CloudAppSecurityDocs/blob/master/CloudAppSecurityDocs/connect-google-apps-to-microsoft-cloud-app-security.md)
* Service: **cloud-app-security**
* GitHub Login: @shsagir
* Microsoft Alias: **shsagir**

**Comment by shsagir** [comment]
Thanks for contacting Microsoft with your suggestion. As we understand it, you're describing a problem with the product configuration guidance and not with the product documentation.

I have forwarded your suggestions to the product team for their consideration. For now I am closing this issue. If you require immediate assistance, please [open a support ticket](https://go.microsoft.com/fwlink/?linkid=2099672).

**Comment by boriskacevich** [comment]
Thanks for the feedback.
The "Location: No organization" is a snapshot glitch that will be fixed, the project should be managed under the organization which is targeted for monitoring and governance. 
Regarding the G Suite API scope - Cloud app security not only monitors connected instances but in many cases also automates the governance process. in G Suite's case, Cloud app security enables user suspension and password change on triggered alerts, this scope is required to enable these scenarios.
